src/main/java/com/denizenscript/depenizen/bukkit/events/breweryx/BreweryModifyBrewScriptEvent.java

```diff
@@ -0,0 +1,68 @@
+package com.denizenscript.depenizen.bukkit.events.breweryx;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.depenizen.bukkit.objects.breweryx.BreweryRecipeTag;
+import com.dre.brewery.api.events.brew.BrewModifyEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class BreweryModifyBrewScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // brewery brew modify
+    //
+    // @Cancellable true
+    //
+    // @Triggers When a Brew has been created or modified.
+    // Usually happens on filling from cauldron, distilling and aging.
+    // Modifications to the Brew or the PotionMeta can be done now
+    // Cancelling reverts the Brew to the state it was before the modification
```

Comment:
Currently this event only really lets you access the data, not modify it - from what I can see Brewery only lets you get and modify the `ItemMeta` from the event (and not set it), so I'm not sure if there's a way to easily implement this in Denizen.
It does look like the event also has a lore modification thing, so can maybe have just that for now?
Otherwise I'd say just leave this event out for the time being, if it's not possible to properly support modifying the brew.